commands/keplr.js

```diff
@@ -281,10 +281,50 @@ const keplr = {
       playwright.keplrWindow(),
       true,
     );
+
+    await playwright.waitForURLLoad(originalURL);
+    await playwright.switchToCypressWindow();
+
+    return true;
```

Comment:
Why is there a need to store the URL at the start and navigate to it once we're about to finish execution?

Reply:
The `switchWallet` test cases are running fine if I comment the first line to get the original URL and  `waitForURLLoad` call.

Reply:
how does it wait for a URL to load if I don't pass any URL?